11-Backbonejs/07-JavaScript-Objects-to-Backbone-Models.md

```diff
@@ -0,0 +1,231 @@
+# Scrabble-JS Total Conversion to Backbone.js!
+
+We've already done Scrabble & Player objects in Javascript.  Now we'll take those generic objects and turn'em into Backbone Models! 
+
+## Learning Goals
+With this live code/lecture you should develop the following skills:
+- Learn how Backbone Models compare with pure Javascript Objects.
+- Learn how to adjust Jasmine tests for Backbone models
+
+
+### Baseline Setup
+The starting code with Scrabble & Player objects are located [here](https://github.com/CheezItMan/scrabble-js-to-backbone).  The file has two JavaScript files `src/scrabble.js` and `src/player.js`.  It also has two spec files `spec/scrabble_spec.js` and `spec/player_spec.js`.  
+
+Go ahead and fork & clone the repo.  
+
+Then take a minute to look at the `scrabble.js` and `player.js` together with your partner to get a bit familiar with them.  
+
+#### Commands
+Because this uses the scaffold you still need to run `npm install` after forking and cloning the repository. **You do not need to run** `npm start` as we won't have a web application.
+
+To run your tests you should use the command `npm test`.  **Please Run your tests before starting to make sure things are working.**
+
+#### Using the REPL
+If you want to interact with the Scrabble objects in the terminal via a REPL, you'll need to do things in a specific way (due to the scaffold's use of webpack).
+
+To access the REPL you should use the command `npm run repl` in your project directory. Once in the REPL, you'll need to use the following syntax to import your file (this example assumes it was named `src/scrabble.js`):
+
+```javascript
+const Scrabble = require('scrabble').default;
+console.log(Scrabble.score("Ada"));
+
+const Player = require('player').default;
+var player1 = new Player("Ada");
+console.log(player1.hasWon());
+```
+
+## Converting Scrabble to a Backbone Model
+
+Scrabble is a fairly simple JavaScript object it has only 3 functions, one of which is a helper function.  `scoreLetter()` and `highestScoreFrom()`.  So we can first Modify it by changing the generic object to instead extend Backbone.Model.
+
+```javascript
+const Scrabble = Backbone.Model.extend( {
+
+});
+```
+
+The argument to extend is a JavaScript object listing attributes and functions that any new instance of the model will have.  The second is optional and we'll skip it here.  
```

Comment:
I think putting "extend" in a code block would make this sentence read more clearly.